mods/bones/init.lua

```diff
@@ -214,21 +216,19 @@ minetest.register_on_dieplayer(function(player)
 	local meta = minetest.get_meta(pos)
 	local inv = meta:get_inventory()
 	inv:set_size("main", 8 * 4)
-	inv:set_list("main", player_inv:get_list("main"))
```

Comment:
are there any stats on how much slower it is to add per item compared to setting the main inventory first?

I imagine it'll be negligible